File: BusReservations/BusReservations.WebAPI/DTOs/ReservationSimpleGetDto.cs
using BusReservations.Core.Domain;

namespace BusReservations.WebAPI.DTOs
{
    public class ReservationSimpleGetDto
    {
        public BusDrivenRouteGetDto BusDrivenRoute { get; set; }
        public SeatDto Seat { get; set; }
        public float FinalSeatPrice { get; set; }
    }
}

File: Frontend/BusReservationsFrontend/src/app/models/reservationSimpleGetDto.ts
import { Bus } from './bus';
import { BusDrivenRoute } from './busDrivenRoute';
import { DrivenRoute } from './drivenRoute';
import { Seat } from './seat';
import { User } from './user';

export interface ReservationSimpleGetDto {
  user: User;
  busDrivenRoute: BusDrivenRoute;
  seat: Seat;
  finalSeatPrice: number;
}
